docs/.gitignore

```diff
@@ -18,3 +18,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```

Comment:
You've added it to gitignore but haven't removed the file from the tree. Adding to gitignore doesn't remove the file